Integration of Boarding control component

mortenterhart opened the pull request.

timkuehnlein committed:
factory

mortenterhart committed:
Include the Boarding Control component to Airport repository

mortenterhart committed:
Included dummy classes, JUnit libraries, tests and factory into the repository

mortenterhart committed:
Merged upstream branches with local branches

mortenterhart committed:
Merge branch 'master' into boarding-control

mortenterhart committed:
Removed out folder from top level directory

patrick246 reviewed (changes requested):
Compiliert nicht, da die POM auf Java 5 gestellt ist. Mit folgendem Code lässt sicht Java 8 einstellen
```
<build>
        <plugins>
            <plugin>
                <artifactId>maven-compiler-plugin</artifactId>
                <configuration>
                    <source>1.8</source>
                    <target>1.8</target>
                </configuration>
            </plugin>
        </plugins>
    </build>
```

mortenterhart committed:
Added dynamic loading of passenger's list out of the database (not implemented yet)

mortenterhart committed:
Set Java 1.8 as SDK for maven

mortenterhart commented:
`pom.xml` ist nun auf 1.8 eingestellt

patrick246 reviewed (changes requested):
Nicht die pom von der component ist das Problem, sondern die von der main-application. Kannst du das bitte noch ändern? Kann Maven bei dir einmal komplett durchlaufen (und zwar das airport-Projekt)?

mortenterhart committed:
Added java version to main-application maven config

mortenterhart committed:
Merge branch 'master' of https://github.com/owels45/Airport

mortenterhart committed:
Merge branch 'master' into boarding-control

mortenterhart committed:
Fixed merge conflict in module ground-operations

patrick246 commented:
Es gibt Konflikte, bitte einmal resolven

mortenterhart committed:
Merge branch 'master' into boarding-control

mortenterhart commented:
Habs gesehen, Konflikte sind gelöst

patrick246 commented:
In deinem Branch fehlt die Klasse CheckInDeskReceipt, die aber im aktuellen master vorhanden ist. Deshalb compiliert es nicht

mortenterhart committed:
Added 1.8 to ground-operations pom.xml

mortenterhart committed:
Merge branch 'master' of https://github.com/owels45/Airport

mortenterhart committed:
Included missing files to the repository

mortenterhart committed:
Updated Boarding Control component with new ground operations interface

mortenterhart committed:
Pulled new master to boarding control and fixed merge conflicts

mortenterhart commented:
Boarding Control kann gemergt werden, bitte kurz Review machen

mortenterhart committed:
Pulled master to boarding control and resolve merge conflicts

mortenterhart committed:
Merge branch 'master' into boarding-control

mortenterhart committed:
Removed dummy classes from BoardingControl component by adding dependencies to maven

mortenterhart committed:
Removed moved dummy base classes from repository

mortenterhart committed:
Corrected BoardingControlReceipt CheckInDesk to Port

mortenterhart committed:
Merge branch 'master' of https://github.com/owels45/Airport into boarding-control

mortenterhart committed:
Merge branch 'master' of https://github.com/owels45/Airport

mortenterhart committed:
Merge branch 'master' into boarding-control

mortenterhart committed:
Adjusted the new path to jar directory in pom.xml

mortenterhart committed:
Changed some settings in pom.xml for BoardingControl

mortenterhart committed:
Added subscriber method for boarding control and amended some missing receipt classes

mortenterhart committed:
Implemented all receiver methods in Airport.java for BoardingControl
Updated BoardingControl component and packed into JAR
Adjusted the POM configuration and outputDirectory path
Removed unnecessary .gitkeep's

filipe96 reviewed (approved):
👨🏻‍💻👍
